static/src/javascripts/projects/commercial/types.d.ts

```diff
@@ -91,8 +91,8 @@ type SlotVisibilityChangedEvent = {
 	slot: Slot;
 };
 
-type GuAdSize = {
+type GuAdSize = Readonly<{
 	width: number;
 	height: number;
-	toString: (_: void) => string;
-};
+	toString: () => string;
```

Comment:
@mxdvl another candidate for template string type...

```suggestion
	toString: () => 'fluid' | `${number},${number}`;
```